lib/ansible/galaxy/collection/gpg.py

```diff
@@ -0,0 +1,298 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2022, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+"""Signature verification helpers."""
+
+from ansible.errors import AnsibleError
+from ansible.galaxy.user_agent import user_agent
+from ansible.module_utils.common.text.converters import to_text
+from ansible.module_utils.urls import open_url
+
+import os
+import subprocess
+import textwrap
+
+from dataclasses import dataclass, fields as dc_fields
+from urllib.error import HTTPError, URLError
+
+
+def run_gpg_verify(b_manifest_file, signature, keyring, display):
```

Comment:
In the light of the efforts to implement mypy linting, maybe add type annotations too?

Reply:
Added